src/hooks/loaders/useCircleLoader.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react'
+import {
+  Easing,
+  Extrapolate,
+  SharedValue,
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated'
+
+type UseCircleLoaderType = {
+  progress: SharedValue<number>
+  animatedStyle: {
```

Comment:
in one place it is called `animatedStyle` in other its called `animatedRotate`. Please be consistent, same with other naming 🙏